application/src/main/kotlin/net/corda/v5/application/messaging/FlowSession.kt

```diff
@@ -83,31 +75,32 @@ interface FlowSession {
      *
      * Note that sessions are closed automatically when the corresponding top-level flow terminates.
      *
-     * So, it's beneficial to eagerly close them in long-lived flows that might have many open sessions that are not needed anymore and
-     * consume resources (e.g. memory, disk etc.).
+     * So, it's beneficial to eagerly close them in long-lived flows that might have many open sessions that are not
+     * needed anymore and consume resources (e.g. memory, disk etc.).
      *
-     * A closed session cannot be used anymore, e.g. to send or receive messages. So, you have to ensure you are calling this method only
-     * when the session is not going to be used anymore. As a result, any operations on a closed session will fail with an
-     * [UnexpectedFlowEndException].
+     * A closed session cannot be used anymore, e.g. to send or receive messages. So, you have to ensure you are calling
+     * this method only when the session is not going to be used anymore. As a result, any operations on a closed
+     * session will fail with an [UnexpectedFlowEndException].
```

Comment:
`[UnexpectedFlowEndException]` this doesn't exist anymore. It currently throws a `CordaRuntimeException`. Should also be included in `@throws`.